kura/org.eclipse.kura.ble.provider/src/main/java/org/eclipse/kura/internal/ble/BluetoothLeServiceImpl.java

```diff
@@ -110,6 +122,12 @@ private boolean startBluetoothUbuntuSnap() {
         }
     }
 
+    private boolean startBluetoothSuppressed() {
+        // Allow to disable the bluetooth service start, e.g. when running inside a container
+        String value = this.systemService.getProperties().getProperty("kura.ble.suppressBluetoothDaemonStart");
+        return Boolean.parseBoolean(value);
```

Comment:
What happens if the property is not defined? Is it better to add a default value instead of return null?

Reply:
I don't think making this more complex makes any sense: https://docs.oracle.com/javase/7/docs/api/java/lang/Boolean.html#parseBoolean(java.lang.String)

> The boolean returned represents the value true if the string argument is not null and is equal, ignoring case, to the string "true".

So "null" is already handled, as "false".

Reply:
That's fine. Could you please take a look at the failing tests?